src/linearmixedmodel.jl

```diff
@@ -673,6 +673,22 @@ function ranef(m::LinearMixedModel{T}; uscale = false, named = false) where {T}
     vnmd
 end
 
+function retbl(mat, trm)
+    merge(
+        NamedTuple{(trm.trm.sym,)}((trm.levels,)),
+        columntable(Tables.table(transpose(mat), header=Symbol.(trm.cnames))),
+        )
+end
+
+"""
+    raneftables(m::LinearMixedModel; uscale = false)
+
+Return the conditional means of the random effects as a NamedTuple of columntables
+"""
+function raneftables(m::LinearMixedModel{T}; uscale = false) where {T}
```

Comment:
```suggestion
function raneftables(m::MixedModel{T}; uscale = false) where {T}
```

This only depends on things that are defined for both LMM and GLMM. (Also, maybe move it to `mixedmodel.jl`? )

Reply:
Good suggestion - now implemented.